tmux.1

```diff
@@ -5567,24 +5567,27 @@ The following keys have a special meaning in the command prompt, depending
 on the value of the
 .Ic status-keys
 option:
-.Bl -column "FunctionXXXXXXXXXXXXXXXXXXXXXXXXX" "viXXXX" "emacsX" -offset indent
-.It Sy "Function" Ta Sy "vi" Ta Sy "emacs"
-.It Li "Cancel command prompt" Ta "q" Ta "Escape"
-.It Li "Delete from cursor to start of word" Ta "" Ta "C-w"
-.It Li "Delete entire command" Ta "d" Ta "C-u"
-.It Li "Delete from cursor to end" Ta "D" Ta "C-k"
-.It Li "Execute command" Ta "Enter" Ta "Enter"
-.It Li "Get next command from history" Ta "" Ta "Down"
-.It Li "Get previous command from history" Ta "" Ta "Up"
```

Comment:
🐞 The current documentation is missing the `j` and `k` keys for the next / previous command in vi-command mode.